docs/source/examples/beam_output.txt

```diff
@@ -1,15 +1,15 @@
 
 Cost
 ----
- 1.473 [m] 
+ 1.62 [m] 
 
 Free Variables
 --------------
-    dx : 0.6667                                          [m]   Length of an element
-     M : [ 1.8e+03   1.42e+03  1.09e+03  800      ... ]  [N*m] Internal moment     
-     V : [ 600       533       467       400      ... ]  [N]   Internal shear      
-\theta : [  -        0.0976    0.174     0.231    ... ]        Slope               
-     w : [  -        0.0325    0.123     0.258    ... ]  [m]   Displacement        
+          dx : 1.2                                             [m]   Length of an element
+     \vec{M} : [ 1.98e+03  1.27e+03  713       317      ... ]  [N*m] Internal moment     
+     \vec{V} : [ 660       528       396       264      ... ]  [N]   Internal shear      
+\vec{\theta} : [  -        0.177     0.285     0.341    ... ]        Slope               
+     \vec{w} : [  -        0.106     0.384     0.759    ... ]  [m]   Displacement        
```

Comment:
It's true! I think the table should be faithful to the varkey rep until such time as we have an established mapping function...